scss/config/_global-settings.scss

```diff
@@ -12,9 +12,102 @@ $decanter-default-settings: (
   breakpoint-helper-selector: #breakpoint-helper,
 );
 
-/// @name Bourbon Settings
-///
-/// @group settings
-$bourbon: (
-  "modular-scale-ratio": 1.333,
+$neat-grid: (
+    columns: $grid-columns,
+    gutter:  $gutter,
+    media: null,
+    color: rgba(#00d4ff, 0.25),
+    direction: ltr,
+);
+
+// Lower boundaries
+// ----------------
+
+/// Screen XS Min
+$screen-xs-min:               0;
```

Comment:
No need to re-declare this breakpoint. It is already included in the configuration from stanford_basic.